### PR TITLE
Changed the footer to fit Asgardeo and added SCIM2 home pages in APIs

### PR DESCRIPTION
**Pull request by himeshsiriwardana**
#### Asgardeo
image

#### IS 7.0.0
image

#### Asgardeo

**Management API docs**
image

**B2B docs**
image

#### IS 7.0.0

**Management API docs**
image

**B2B docs**
image

### CODE REVIEW

**Review by brionmario — APPROVED** [review]

File: en/identity-server/7.0.0/mkdocs.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ site_url: https://is.docs.wso2.com/en/7.0.0
 extra:
   base_path: /en/7.0.0
   is_version: 7.0.0
+  footer:
```
Comment (pavinduLakshan) [inline en/identity-server/7.0.0/mkdocs.yml:25]: Let's also add stackoverflow link for Asgardeo.

https://stackoverflow.com/questions/tagged/wso2-asgardeo

https://github.com/himeshsiriwardana/docs-is/blob/fixes-07-15/en/asgardeo/mkdocs.yml

File: en/asgardeo/docs/apis/restapis/scim2-sp-configs.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      properties:
+        status:
+          type: string
+          example: "404"
+        schemas:
+          type: string
+          example: urn:ietf:params:scim:api:messages:2.0:Error
+        detail:
+          type: string
+          example: Schema not found.
```
Comment (AnuradhaSK) [inline en/asgardeo/docs/apis/restapis/scim2-sp-configs.yaml:106]: missing a new line EoF

Comment (AnuradhaSK) [reply en/asgardeo/docs/apis/restapis/scim2-sp-configs.yaml:106]: Check other yaml files as well

**Review by AnuradhaSK — APPROVED** [review]

File: en/asgardeo/docs/apis/restapis/scim2-sp-configs.yaml (hunk 2 above)
Comment (himeshsiriwardana) [reply en/asgardeo/docs/apis/restapis/scim2-sp-configs.yaml]: Fixed with https://github.com/wso2/docs-is/pull/4661/commits/926725dfe60186efc054f2dcddbec1014c488729

**Review by AnuradhaSK — APPROVED** [review]

**Pull request merged**